sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -69,7 +69,7 @@ with tracer.start_as_current_span(name="MyApplication"):
     client.create_container('mycontainer')  # Call will be traced
 ```
 
-Azure Exporter can be found in the package `opentelemetry-azure-monitor-exporter`
+Azure Exporter can be found in the [package](https://github.com/microsoft/opentelemetry-exporters-python) `opentelemetry-azure-monitor-exporter`
```

Comment:
Shouldn't we link to the package on PyPi? It will in turn link back to source if customers want access to that...

Reply:
E.g. https://pypi.org/project/opentelemetry-azure-monitor-exporter/